### ISSUE TITLE
Revision of lecture/exercise bubble path classification

### ISSUE THREAD

**Issue by AndreWeiner** [AndreWeiner/ml-cfd-lecture#27]
Hi @JanisGeise,
I just updated the classification exercise (exercise 6). Please let me know if changes/additions are required. I uploaded a reference solution to your cloud folder.
Thanks!
Andre

**Comment by JanisGeise** [comment]
Hi @AndreWeiner,

the exercise worked without any problems. Recently, I get warnings from PyTorch regarding Cuda, maybe (in case other people might get this warning too) you can add a note to ignore such warnings, since the training is done on CPU. 

Out of curiosity, in the last cell of the lecture notebook (cell 90) you add *0.01* to the predictions (`prediction = pt.argmax(prob, dim=1) + 0.01)`,  is this just for plotting purposes?

Regards
Janis

**Comment by AndreWeiner** [comment]
Perfect, thanks for the feedback. I got the warning about Cuda, too. Not sure what it is about. I thought it was only related to my system.
Yes, the little increment is just for plotting - I think to avoid the influence of roundoff errors in the Matplotlib ColorMap. In the current version of the notebook, it also works for me without adding the increment.
Thanks again!
Andre

**Comment by JanisGeise** [comment]
Ok, it's good to know that this seems not to be just an issue with my torch installation / setup. Thanks for the explanation regarding the increment.

Regards,
Janis
